### DISCUSSION TITLE
Is there the possibility of being able to extract the data of steps and heart rate from the zepp application?

### DISCUSSION THREAD

**Discussion by JosueButron** [zepp-health/zeppos-forum#203] [General]
I am working on a project in which I need to collect this data and I thought to use the xiaomi band 7 devices that can be connected with the zepp app, however I do not know if there is a way in which I can extract this information through an API or how to could do ?

**Comment by liqin-ZeppHealth** [comment]
we don't have an API on cloud to extract user data now, you can only get data on the device by mini-app, you can check sensor APIs of health data for mini-app
